tests/tests_integration/test_api/test_sequences_data.py

```diff
@@ -171,7 +167,7 @@ def test_delete_range(self, new_seq_long):
             column_external_ids=new_seq_long.column_external_ids, rows=data, id=new_seq_long.id
         )
         COGNITE_CLIENT.sequences.data.delete_range(start=4, end=15, id=new_seq_long.id)
-        dps = COGNITE_CLIENT.sequences.data.retrieve(start=0, end=None, id=new_seq_long.id)
+        dps = COGNITE_CLIENT.sequences.data.retrieve(start=0, end=None, id=new_seq_long.id)  # noqa F841
```

Comment:
no noqa comments please, can just fix dps = if you want

Reply:
I added it since `dps` is used in the out-commented lines below. Removing `noqa`, but keeping assignment to `dps`.